KDAB/knut
# refactor!: Change the test API for scripts

This change generalize the `test` function for script, allowing to have multiple test functions.
Also remove the Knut.Test module, not needed anymore.

This makes the whole testing more streamline, and easier to work with.

- remove the `Knut.Test` module and `TestCase` item
- add `compare` and `verify` to `Script` and `ScriptDialog`
- run all methods starting with `test_` in test mode
- don't run `test` anymore (breaking change here), as it does not make sense to have a special case for this.

Related to #49
